packages/synapse-react-client/test/lib/containers/dataaccess/AccessHistoryDashboard.test.tsx

```diff
@@ -142,39 +159,72 @@ describe('AccessHistoryDashboard tests', () => {
     )
   })
 
-  it('Updates the passed props and URLSearchParams when updating user/team name', async () => {
-    const { history } = renderComponent()
+  it('Renders only the submission table for non-ACT users', async () => {
```

Comment:
New test is heavily based on the one known to be flaky, so there's some risk in adding it.